packages/botonic-plugin-text-classification/src/process/text-processor.ts

```diff
@@ -8,7 +14,21 @@ export class TextProcessor {
   ) {}
 
   process(text: string): { sequence: string[]; input: Tensor2D } {
-    //TODO: pending to be implemented
-    return { sequence: null, input: null }
+    const sequence = this.preprocessor.preprocess(text, PADDING_TOKEN)
+    const maskedSequence = this.maskUnknownTokens(sequence)
+    const encodedSequence = this.encodeSequence(maskedSequence)
+    const input = tensor([encodedSequence]) as Tensor2D
+    return { sequence, input }
+  }
+
+  private maskUnknownTokens(sequence: string[]): string[] {
```

Comment:
here we can't use IndexItems.maskUnknownTokens right?
what about creating a Vocabulary class which just contains an indexedTokens field of type IndexedItem and add a  maskUnknownTokens method?
I guess it's better to do it after you merge this & the indexedItems PR

Reply:
In the future we can use `IndexedItems.maskUnknownItems` method. But first, we need to merge the PR where it is implemented. This would be implemented after both PR are merged. 👍🏽